src/main/java/com/celements/rights/access/DefaultRightsAccessFacade.java

```diff
@@ -79,4 +79,17 @@ public boolean hasAccessLevel(EntityReference ref, EAccessLevel level, XWikiUser
         user, ret);
     return ret;
   }
+
+  @Override
+  public boolean isLoggedIn() {
+    boolean ret;
+    if (!"XWiki.XWikiGuest".equals(getContext().getUser())) {
```

Comment:
Please use XWikiRightService.GUEST_USER_FULLNAME instead of hard coded String (same applies in the tests)